<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QgsProjectionSelectionWidget" name="mQgsProjectionSelectionWidget">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>160</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsDoubleSpinBox" name="mQgsDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QgsDoubleSpinBox" name="mQgsDoubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>300</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Effect</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectcrs">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>        SELECT CRS</string>
   </property>
  </widget>
  <widget class="QLabel" name="circlesize">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>     CIRCLE SIZE</string>
   </property>
  </widget>
  <widget class="QLabel" name="update">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE INTERVAL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> SELECT LAYER</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSaveOption">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>380</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Save as temporary layer</string>
   </property>
   <item>
    <property name="text">
     <string>Save as temporary layer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Save as file</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="seconds">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>310</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="meters">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>240</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>meters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maskSavePath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>file location (A temporary layer will be created)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>29</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GROUP NAME</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BOTTOM LAYER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TOP LAYER(S)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bottom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="top">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
